src/main/java/seedu/address/logic/commands/ListBuyerCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.displayable.buyer.Buyer;
+
+/**
+ * Displays information of a specified buyer.
+ */
+public class ListBuyerCommand extends Command {
```

Comment:
Is there any reason why the name of the command was changed from ListBuyersCommand to ListBuyerCommand?

Reply:
Lists 1 particular buyer only